doc/source/serve/index.md

```diff
@@ -121,9 +126,89 @@ Because it's built on top of Ray, you can run it anywhere Ray can: on your lapto
+
+:::
+
+:::{dropdown} ML Platform engineer
+:animate: fade-in-slide-down
+
+Serve specializes in ML model serving and can be a reliably component in your ML Platform stack.
+Serve supports arbitrary Python code and therefore integrates well with the MLOps ecosystem. You can use it with model optimizers (ONNX, TVM), model monitoring systems (Seldon Alibi, Arize), model registries (MLFlow, Weights and Biases), machine learning frameworks (XGBoost, Scikit-learn), data app UIs (Gradio, Streamlit), and Web API frameworks (FastAPI, gRPC).
+
+:::
+
+
+## How does Serve compare to X
+
+:::{dropdown} TFServing, TorchServe, ONNXRuntime
+:animate: fade-in-slide-down
+
+Ray Serve is *framework agnostic*, you can use any Python framework and libraries.
+We believe data scientists are not bounded a particular machine learning framework.
+They use the best tool available for the job.
+
+Compared to these framework specific solution, Ray Serve doesn't perform any optimizations
+to make your ML model run faster. However, you can still optimize the models yourself
+and run them in Ray Serve: for example, you can run a model compiled by
+[PyTorch JIT](https://pytorch.org/docs/stable/jit.html).
+:::
+
+:::{dropdown} AWS SageMaker, Azure ML, Google Vertex AI
+:animate: fade-in-slide-down
+
+Ray Serve brings the scalability and parallelism of these hosted offering to
+your own infrastructure. You can use our [cluster launcher](cluster-cloud)
+to deploy Ray Serve to all major public clouds, K8s, as well as on bare-metal, on-premise machines.
+
+Compared to these offerings, Ray Serve lacks a unified user interface and functionality
+let you manage the lifecycle of the models, visualize it's performance, etc. Ray
+Serve focuses on just model serving and provides the primitives for you to
+build your own ML platform on top.
+:::
+
+:::{dropdown} Seldon, KServe, Cortex
+:animate: fade-in-slide-down
+
+You can develop Ray Serve on your laptop, deploy it on a dev box, and scale it out
+to multiple machines or K8s cluster without changing one lines of code. It's a lot
+easier to get started with when you don't need to provision and manage K8s cluster.
+When it's time to deploy, you can use the our native Kubernetes Operator
+to transparently put your Ray Serve application in K8s.
+:::
+
+:::{dropdown} BentoML, Comet.ml, MLflow
+:animate: fade-in-slide-down
+
+Ray Serve is a special purpose distributed model server built for large scale applications.
+This means we can work with any model packaging and registry format. Many of the tools are
+focused on serving one models. Ray Serve is built for end-to-end machine learning application
+in mind with our unique model composition API and advanced autoscaling capabilities.
```

Comment:
Many of the tools are focused on serving one ML framework-specific type of a model. By contrast, Ray Serve is framework-agnostic and allows you to build an end-to-end machine learning application with its unique model composition API and advanced autoscaling capabilities.